blueocean-core-js/src/js/utils/UrlUtils.js

```diff
@@ -34,7 +34,10 @@ export const buildPipelineUrl = (organization, fullName, tabName) => {
 
     return tabName ? `${baseUrl}/${tabName}` : baseUrl;
 };
-
+export const rootPathCreate = () => {
```

Comment:
I'm really a bit confused why we have this monolithic URL handling file, why not just make the right URLs where they're needed? Easier to maintain, and would not require a re-publish of core-js...